dotcom-rendering/src/components/Island.test.tsx

```diff
@@ -348,11 +348,16 @@ describe('Island: server-side rendering', () => {
 	test('SetABTests', () => {
 		expect(() =>
 			renderToString(
-				<SetABTests
-					isDev={false}
-					pageIsSensitive={false}
-					abTestSwitches={{}}
-				/>,
+				<ConfigProvider
```

Comment:
Added this to make the test pass here. I'm assuming we don't need to explicitly add a `ConfigProvider` to where the component is rendered because it is already available globally? But would be good to confirm how this works.

Reply:
Yes that’s exactly right!

Reply:
Ok perfect, thanks